### PR TITLE
5.0.0 blog post

### PR DESCRIPTION
**Pull request by devlead**

### CODE REVIEW

File: input/blog/2024-11-12-cake-v5.0.0-released.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+### Highlights of this release
+
+- .NET 9 support & C# 13 support
+- FreeBSD support
+- Addin recommended Cake version set to 5.0.0
+- 100 million downloads of Cake.Tool
+
+
+### .NET 9 support & C# 13 support
```
Comment (pascalberger) [inline input/blog/2024-11-12-cake-v5.0.0-released.md:20]: Should explicitly mention that we no longer support .NET 6 and .NET 7?

Something like we did in the Cake 3.0 blog post: https://cakebuild.net/blog/2022/11/cake-v3.0.0-released

File: input/blog/2024-11-12-cake-v5.0.0-released.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+```
+
+The recommended target framework monikers for addins are now
+
+- `net9.0`
+- `net8.0`
+
+### 100 million downloads of Cake.Tool
+
+We are thrilled to announce that Cake.Tool has reached an incredible milestone of 100 million downloads on NuGet.org! This accomplishment wouldn't have been possible without the contributions of over 300 individuals and the unwavering support of our users. Join us as we reflect on the journey of Cake, featuring 129 releases and over 300 third-party community add-ins.
```
Comment (pascalberger) [inline input/blog/2024-11-12-cake-v5.0.0-released.md:54]: Make this a link to https://cakebuild.net/community/thanks/#contributors ?

**Review by pascalberger — APPROVED** [review]
Some smaller suggestions, but other LGTM

File: input/blog/2024-11-12-cake-v5.0.0-released.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+This won't break the build, but you might see warnings like the one below when addins or modules are loaded.
+
+```bash
+The assembly 'Cake.Addin, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null'
+is referencing an older version of Cake.Core (4.0.0).
+For best compatibility it should target Cake.Core version 5.0.0.
+```
+
+The recommended target framework monikers for addins are now
```
Comment (gep13) [inline input/blog/2024-11-12-cake-v5.0.0-released.md:47]:
```suggestion
The recommended target framework monikers for addins are now:
```

File: input/blog/2024-11-12-cake-v5.0.0-released.md (hunk 2 above)
Comment (gep13) [inline input/blog/2024-11-12-cake-v5.0.0-released.md:54]:
```suggestion
We are thrilled to announce that [Cake.Tool](https://www.nuget.org/packages/Cake.Tool) has reached an incredible milestone of 100 million downloads on NuGet.org! This accomplishment wouldn't have been possible without the [contributions of over 300 individuals](https://cakebuild.net/community/thanks/#contributors) and the unwavering support of our users. Join us as we reflect on the journey of Cake, featuring 129 releases and over 300 third-party community add-ins.
```

**Review by gep13 — CHANGES_REQUESTED** [review]

File: input/blog/2024-11-12-cake-v5.0.0-released.md (hunk 1 above)
Comment (devlead) [reply input/blog/2024-11-12-cake-v5.0.0-released.md:20]: Added this blurb

> Cake itself will no longer run on .NET 6 or .NET 7, but Cake will continue to support the building of .NET Framework projects, as well as projects targeting .NET 7.0 or older.

**Review by nils-a — APPROVED** [review]
LGTM!

**Pull request merged**